src/sst/elements/vanadis/decoder/vriscv64decoder.h

```diff
@@ -128,15 +128,13 @@ class VanadisRISCV64Decoder : public VanadisDecoder
     }
 
 
-    void tick(SST::Output* output, uint64_t cycle) override
+    void tick(SST::Output* output, uint64_t) override
```

Comment:
Are we better off changing the tick call rather than leaving the uint64_t?

Reply:
I wasn't sure how invasive to be here. There's no reason for the cycle counter in the tick here for RISC-V in the patch... `cycle` is implemented in the register file, but I wasn't sure what you wanted to do for MIPS (it's not used their either). Removing it wouldn't impact the current code.

Reply:
We should just change the tick() function. However, I'm ok leaving this and fixing it after.